test/container/container_tests.cpp

```diff
@@ -102,4 +104,65 @@ TEST_F(ContainerTestSuite, HandleEvent)
 
   // Then
   EXPECT_TRUE(called);
-}
\ No newline at end of file
+}
+
+TEST_F(ContainerTestSuite, Sdk)
+{
+  std::string filePath = "testDataDir/frame_list/";
+
+  auto result = m_sut->load(filePath);
+  EXPECT_TRUE(result);
+
+  auto sdk = m_sut->sdk();
+  {
+    const auto& framesJsonString = sdk->getFramesInfo();
+
+    auto frames = nlohmann::json::parse(framesJsonString);
+    EXPECT_EQ(frames.size(), 3);
+  }
+
+  {
+    const auto index = sdk->currentFrameIndex();
+    EXPECT_EQ(index, 0);
+  }
+
+  {
+    auto success = sdk->setCurrentFrame("#invalidFrameName");
+    EXPECT_FALSE(success);
+  }
+  {
+    auto success = sdk->setCurrentFrame("Frame 2");
+    EXPECT_TRUE(success);
+
+    const auto index = sdk->currentFrameIndex();
+    EXPECT_EQ(index, 1);
+  }
+
+  {
+    const auto launchFrameIndex = sdk->launchFrameIndex();
+    EXPECT_EQ(launchFrameIndex, 0);
+  }
+  {
+    auto success = sdk->setLaunchFrame("#invalidFrameName");
+    EXPECT_FALSE(success);
+  }
+  {
+    auto success = sdk->setLaunchFrame("Frame 2");
+    EXPECT_TRUE(success);
+
+    const auto index = sdk->launchFrameIndex();
+    EXPECT_EQ(index, 1);
+  }
+
+  {
+    const auto& fontsJsonString = sdk->requiredFonts();
+
+    auto fonts = nlohmann::json::parse(fontsJsonString);
+    EXPECT_EQ(fonts.size(), 1);
```

Comment:
String comparison may be better? So user can identify what `requiredFonts` returns actually.

Reply:
Fixed